### PR TITLE
Adding article links by community members

### PR DESCRIPTION
**Pull request by AyDeveloper**
List of useful articles published on hashnode

### CODE REVIEW

**Review by kjhman21 — COMMENTED** [review]
Thanks a lot!

**Review by kjhman21 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -131,6 +131,19 @@ This page introduces all links related to core and ecosystem development of Klay
...
+
+* [Building with Klaytn](https://oxpampam.hashnode.dev/building-with-klaytn-what-is-klaytn-and-how-to-get-started-as-a-developer)
+* [Beginner’s Guide To Kaikas Wallet](https://oxpampam.hashnode.dev/the-beginners-guide-to-kaikas)
+* [Create and Deploy your First Smart Contract with Klaytn IDE](https://oxpampam.hashnode.dev/create-and-deploy-your-first-smart-contract-with-klaytn-ide)
+* [How To Create and Deploy an ERC20 Token on Klaytn](https://oxpampam.hashnode.dev/how-to-create-and-deploy-an-erc20-token-on-klaytn))
+* [Building a Klaytn bApp with Caver-Js](https://oxpampam.hashnode.dev/building-a-klaytn-bapp-with-caver-js)
+* [How to set up an Hardhat project for Klaytn](https://oxpampam.hashnode.dev/how-to-set-up-a-hardhat-project-for-klaytn)
+* [Klaytn Baobab Faucet: Solidity Smart Contract](https://oxpampam.hashnode.dev/klaytn-baobab-faucet-solidity-smart-contract)
+* [Beginner's guide on how to add Klaytn to Metamask
+](https://oxpampam.hashnode.dev/beginners-guide-on-how-to-add-klaytn-to-metamask)
```
Comment (kjhman21) [inline README.md:144]:
```suggestion
* [Beginner's guide on how to add Klaytn to Metamask](https://oxpampam.hashnode.dev/beginners-guide-on-how-to-add-klaytn-to-metamask)
```

**Review by kjhman21 — CHANGES_REQUESTED** [review]
Need to fix to remove a line break.

**Review by terrikwak — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AyDeveloper) [reply README.md:144]: Alright would fix that ASAP

**Review by kjhman21 — APPROVED** [review]

**Pull request merged**